repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml

```diff
@@ -0,0 +1,393 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:update/hippo:registry/DW-1284 - Migration from publication to series:
+      hipposys:batchsize: 10
+      hipposys:description: ''
+      hipposys:dryrun: false
+      hipposys:loglevel: TRACE
+      hipposys:parameters: "{\r\n    \"inputFile\" :  \"/content/assets/assetsmiguel/seriesinfoexport2.json/seriesinfoexport2.json/hippogallery:asset\"\
+        \ \r\n}"
+      hipposys:query: /jcr:root/content/documents/corporate-website//*[(@jcr:primaryType='publicationsystem:series')]
+      hipposys:script: "package org.hippoecm.frontend.plugins.cms.admin.updater\r\n\
```

Comment:
This inline script should be moved to its own file.

Reply:
There is a DW-1284---Migration-from-publication-to-series.groovy file, is that what you mean?

Should I then remove the hipposys:script field and content in the yaml file?

Reply:
Yes please.

Reply:
You still need a YMAL file. You need one YMAL and one Groovy file per script.

Reply:
Ok, it is done. We still have to squash the changes, I will do it later.